### DISCUSSION TITLE
wrong translation for chinese

### DISCUSSION THREAD

**Discussion by vivisol** [vercel/next.js#62262] [Help]
### Summary

these is a wrong translation for chinese here [https://www.nextjs.cn/learn/basics/create-nextjs-app](url)
The correction is as follows:
![image](https://github.com/vercel/next.js/assets/22468237/1eaf576a-f30e-42b1-987c-06a6f69322ed)

**Comment by wwwdepot** [comment]
see https://github.com/vercel/next.js/issues/38672#issuecomment-1185346153
